pkg/model/context.go

```diff
@@ -183,7 +183,7 @@ func (m *KopsModelContext) NodeInstanceGroups() []*kops.InstanceGroup {
 
 // CloudTagsForInstanceGroup computes the tags to apply to instances in the specified InstanceGroup
 func (m *KopsModelContext) CloudTagsForInstanceGroup(ig *kops.InstanceGroup) (map[string]string, error) {
-	labels := make(map[string]string)
+	labels := m.CloudTags(ig.Name, false)
```

Comment:
Line 212 is now redundant. Should pass `m.AutoscalingGroupName(ig)` instead of `ig.Name` to make line 213 redundant.

Reply:
I'm trying to figure out how the `Name` tag is being set correctly without using `m.AutoscalingGroupName(ig)` here.

Reply:
OK, changed to m.AutoscalingGroupName(ig) to not complicate things.